### PR TITLE
Draft of the checklist api

### PR DESCRIPTION
**Pull request by allancalix**

### CODE REVIEW

File: todo/todo.proto [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+message UpdateChecklistRequest {
+  // The id of the checklist to update.
+  string id = 1;
+  // The content of the updated checklist.
+  ChecklistContent content = 2;
+}
+
+message DeleteChecklistRequest {
+  // The id of the checklist to update.
+  string id = 1;
```
Comment (jafow) [inline todo/todo.proto:72]: id is type string here and in the `GetChecklistRequest` [is type int](https://github.com/jafow/tododb/pull/1/files#diff-16e87438b0d091e4c6e968f612e33db3R60). is it on purpose or a typo? (more like type-o, eh?)

File: todo/todo.proto [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+message Item {
+  // A unique identifier for a checklist todo item.
+  string id = 1;
+  // The display text for an item.
+  string description = 2;
+  // The current state of the item (i.e. is the box checked).
+  bool complete = 3;
+  // The last time an item was updated in seconds since epoch. Not relevant for draft checklists..
+  google.protobuf.Timestamp last_update_time = 4;
```
Comment (jafow) [inline todo/todo.proto:132]: cool

**Comment by jafow** [comment]
@allancalix IMO I think we should move this into a proper repo -- fine to keep it here but i only made this to put the sql in :tada:

File: todo/todo.proto (hunk 1 above)
Comment (allancalix) [reply todo/todo.proto:72]: Indeed, this is a type-O!

**Comment by allancalix** [comment]
Sounds good, mostly I just wanted the PR for review

**Pull request merged**